### PR TITLE
[DEV] Feature/azure integration

### PR DESCRIPTION
**Pull request by MatheusLeall**
# Infos

[Task on sprint board](https://juntossomosmais.monday.com/boards/1521247929/pulses/4964788142)

## What is being delivered?

- Adapt lib to jsm django-template
- Implement CI/CD

## What impacts?

No impacts

## Reversal plan

Revert PR changes

## Where to monitor

Pipy page

---
Doubts about the flow?
[Take a look at JS+ gitflow!](https://github.com/juntossomosmais/gitflow/blob/main/gitflow/backend/README.md)

Doubts about the code standards?
[Take a look at JS+ python playbook!](https://github.com/juntossomosmais/playbook/blob/master/backend/python.md)

### CODE REVIEW

File: tests/unit/test_app.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def test_sum():
```
Comment (dhiulysoares) [inline tests/unit/test_app.py]: [question] why this test exists?

Comment (MatheusLeall) [reply tests/unit/test_app.py]: Just to help us configure the pipeline correctly, will be removed in the future

File: docker-compose.yml [hunk 2]
```diff
@@ -120,3 +134,6 @@ services:
 
 volumes:
   s3-folder: null
+
+networks:
+  importexportstompnet:
```
Comment (niltonfrederico) [inline docker-compose.yml]: [question]
Why add a custom network?

Comment (MatheusLeall) [reply docker-compose.yml]: To follow our django-template, does make sense?

Comment (MatheusLeall) [reply docker-compose.yml]: Removed!

File: import_export_stomp/tasks.py [hunk 3]
```diff
@@ -180,10 +173,8 @@ def before_import_row(self, row, **kwargs):
     import_job.save()
 
 
-@shared_task(bind=False)
-def run_import_job(pk, dry_run=True):
-    log.info(f"Importing {pk} dry-run {dry_run}")
-    import_job = models.ImportJob.objects.get(pk=pk)
+def run_import_job(import_job, dry_run=True):
+    logger.info(f"Importing {import_job.pk} dry-run {dry_run}")
```
Comment (mairaafreitas) [inline import_export_stomp/tasks.py:177]:
```suggestion
    logger.info("Importing %s dry-run %s", import_job.pk, dry_run)
```

**Review by mairaafreitas — APPROVED** [review]

**Review by mairaafreitas — COMMENTED** [review]
Missing type hint in tests

File: tests/unit/test_admin_actions.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    return ExportJob.objects.create(
+        app_label="app_label",
+        model="model",
+        queryset="[]",
+        site_of_origin="http://example.com",
+    )
+
+
+@pytest.mark.django_db
+def test_run_export_job_action(admin_site, export_job, user):
```
Comment (mairaafreitas) [inline tests/unit/test_admin_actions.py]: [suggestion]
Add test inside a class

**Review by danilocastelhano1 — APPROVED** [review]

**Pull request merged**